cloud/gcp/main.tf

```diff
@@ -0,0 +1,32 @@
+provider "google" {
+ alias = "impersonation"
+ scopes = [
+   "https://www.googleapis.com/auth/cloud-platform",
+   "https://www.googleapis.com/auth/userinfo.email",
+ ]
+}
+
+data "google_service_account_access_token" "default" {
+ provider               	= google.impersonation
+ target_service_account 	= var.terraform_service_account
+ scopes                 	= ["userinfo-email", "cloud-platform"]
+ lifetime               	= "1200s"
+}
+
+provider "google" {
+ project 		= "civiform-demo"
+ access_token	= data.google_service_account_access_token.default.access_token
+ request_timeout 	= "1200s"
```

Comment:
thought (non-blocking): 20m seems like quite a long time for a request timeout. consider lowering if appropriate

Reply:
Done